cmd/storagenode/main.go

```diff
@@ -319,6 +390,42 @@ func isOperatorWalletValid(wallet string) error {
 	return nil
 }
 
+// clr clears the screen so it can be redrawn
+func clr() {
+	var clear = make(map[string]func())
+	clear["linux"] = func() {
```

Comment:
Interestingly, I took a look at the color library you're using, and it is actually _interpreting_ escape codes and translating them to native Windows calls as appropriate! If you write that sequence to `color.Output`, it should work on Windows.

But ok sorry I'll leave you alone :D

Reply:
Oh, I actually just assumed that library wasn't going to work on Windows, but that's good to know. I'll get this PR merged, but I can submit a new PR later on to use the actual escape sequences later on.

Reply:
👍